src/main/java/com/google/sps/servlets/CronServlet.java

```diff
@@ -112,9 +113,10 @@ public void resetPostedYet(DatastoreService datastore) {
             String description = (String) deed.getProperty(DESCRIPTION);
             String posted_yet_string = (String) deed.getProperty(POSTED_YET);
             boolean posted_yet_bool = Boolean.parseBoolean(posted_yet_string);
+            String link = (String) deed.getProperty(LINK);
             long timestamp = (long) deed.getProperty(TIME_STAMP);
  
-            GoodDeed deedObject = new GoodDeed(key, id, title, description, posted_yet_bool, timestamp);
+            GoodDeed deedObject = new GoodDeed(key, id, title, description, posted_yet_bool, timestamp, link);
```

Comment:
Do we need to handle the exception thrown in the constructor here?

Reply:
Possible null values are handled in GoodDeedsServlet